spec/javascripts/resources/InventoryInvitableSpec.js

```diff
@@ -0,0 +1,20 @@
+describe('Resource: InventoryInvitable', function() {
+  var subject, scope, $httpBackend;
+
+  beforeEach(module('PDRClient'));
+
+  beforeEach(inject(function($resource, $rootScope, InventoryInvitable, $injector) {
+    scope = $rootScope.$new();
+    subject = InventoryInvitable;
+    $httpBackend = $injector.get('$httpBackend');
+  }));
+
+  it("returns all invitables users for an inventory", function(){
+    $httpBackend.expectGET('/v1/inventories/10/invitables').respond([1, 2]);
+    var users = subject.list({
+      inventory_id: 10
+    });
+    $httpBackend.flush();
+    expect(users.length).toEqual(2);
```

Comment:
If I change what I get back from that `$httpBackend` call, I'll get back a different result, and this expectation will fail.  Essentially, `$httpBackend` is mocking out the response, so it's not indicative at all of what we're going to get back.

What would be a better test would be to ensure that it hits the right route.  I believe that the `expectGET` is sufficient to cover that case.

You would also want to be sure there are no outstanding expectations or outstanding requests after the test has run (through $httpBackend.verifyNoOutsandingExpectation()` and `$httpBackend.verifyNoOutsandingRequest()`).